packages/console/src/log/LogView.tsx

```diff
@@ -96,15 +103,29 @@ class LogView extends PureComponent {
     }
   }
 
-  componentWillUnmount() {
+  componentWillUnmount(): void {
     this.stopFlushTimer();
     this.stopListening();
     this.destroyMonaco();
 
     window.removeEventListener('resize', this.handleResize);
   }
 
-  getMenuActions(shownLogLevels) {
+  cancelListener?: () => void | null;
+
+  editor?: monaco.editor.IStandaloneCodeEditor;
+
+  editorContainer: HTMLDivElement | null;
```

Comment:
```suggestion
  editorContainer?: HTMLDivElement;
```

Reply:
EditorContainer is a ref, so it's value can be null